script/upgrade/GoerliM2Upgrade.s.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/presets/ERC20PresetFixedSupply.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
+
+import "../../src/contracts/interfaces/IETHPOSDeposit.sol";
+import "../../src/contracts/interfaces/IBeaconChainOracle.sol";
+
+import "../../src/contracts/core/StrategyManager.sol";
+import "../../src/contracts/core/Slasher.sol";
+import "../../src/contracts/core/DelegationManager.sol";
+
+import "../../src/contracts/strategies/StrategyBaseTVLLimits.sol";
+
+import "../../src/contracts/pods/EigenPod.sol";
+import "../../src/contracts/pods/EigenPodManager.sol";
+import "../../src/contracts/pods/DelayedWithdrawalRouter.sol";
+
+import "../../src/contracts/permissions/PauserRegistry.sol";
+import "../../src/contracts/middleware/BLSPublicKeyCompendium.sol";
+
+import "../../src/test/mocks/EmptyContract.sol";
+import "../../src/test/mocks/ETHDepositMock.sol";
+
+import "forge-std/Script.sol";
+import "forge-std/Test.sol";
+
+// # To load the variables in the .env file
+// source .env
+
+// # To deploy and verify our contract
+// forge script script/upgrade/GoerliM2Deployment.s.sol:GoerliM2Deployment --rpc-url $RPC_URL  --private-key $PRIVATE_KEY --broadcast -vvvv
+contract GoerliM2Deployment is Script, Test {
+    Vm cheats = Vm(HEVM_ADDRESS);
+
+    string public deploymentOutputPath = string(bytes("script/output/M1_deployment_goerli_2023_3_23.json"));
```

Comment:
update this file?

Reply:
This path is to get the addresses from the M1 deployment

The output of this script is getting written to: `script/output/M2_deployment_data.json`